api-server/README.md

```diff
@@ -0,0 +1,112 @@
+## Mintlayer API server
+
+### Introduction
+
+The Mintlayer API server is a tool that scans the blockchain and publishes its data in a database for easy access. Technically speaking, this is done to achieve the trade-off where the blockchain itself contains the minimal required amount of data, while the API server indexes all the data for easy reach. The API server is used by block explorers and web wallets. The API server is made to be usable by anyway, such as exchanges, people interested in writing tooling for the blockchain, or anything else.
```

Comment:
This seems a bit weird:
The API server is made to be usable by **anyway**
probably should be "by anyone" or "in many ways"?

Reply:
You're right. Fixed. Thanks.